### PR TITLE
CLIENT-2145

### PR DESCRIPTION
**Pull request by DomPeliniAerospike**

### CODE REVIEW

**Review by justinlee-aerospike — CHANGES_REQUESTED** [review]
Generally looks good.  I'll approve after making the two discussed changes and also verifying that the typings specified in index.d.ts work for Typescript.

File: scripts/prebuiltBinding.js [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+  const words = stdout.split(' ')
+  const lastToken = words[5]
+  const OS = lastToken.split('\n')[1]
+  const arch = lastToken.split('\n')[2]
+  const nodeVersion = lastToken.split('\n')[3].split('.')[0]
+  if ((words[1][0] === '3') && (words[0] !== 'LibreSSL')) {
+    if (OS === 'Linux') {
+      if (arch === 'x86_64') {
+        if (nodeVersion === 'v19') {
+          fs.rename('lib/binding/openssl@3/node-v111-linux-x64', 'lib/binding/node-v111-linux-x64', (err) => {
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply scripts/prebuiltBinding.js]: This is covered in my recent commit: https://github.com/aerospike/aerospike-client-nodejs/pull/504/commits/014cd8448ce3f7000adfb8fc1cdef5eebbdf2f48 
Resolving now.

File: test/batch_write.js [hunk 2]
```diff
@@ -61,16 +61,16 @@ describe('client.batchWrite()', function () {
       const batchRecords = [
         {
           type: batchType.BATCH_READ,
-          key: new Key(helper.namespace, helper.set, 'test/batch_write/1'),
+          key: new Key(helper.namespace, helper.set, 'test/batch_write/2'),
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply test/batch_write.js]: This is covered in my recent commit: https://github.com/aerospike/aerospike-client-nodejs/pull/504/commits/014cd8448ce3f7000adfb8fc1cdef5eebbdf2f48 
Resolving now.

File: src/main/cdt_ctx.cc [hunk 3]
```diff
@@ -22,9 +22,27 @@
 
 extern "C" {
 #include <aerospike/as_cdt_ctx.h>
+#include <aerospike/as_cdt_order.h>
 }
 
 using namespace v8;
+enum ctx_type 
+{
```
[the remark this answers is not in the document]
Comment (DomPeliniAerospike) [reply src/main/cdt_ctx.cc]: This is covered in my recent commit: https://github.com/aerospike/aerospike-client-nodejs/pull/504/commits/014cd8448ce3f7000adfb8fc1cdef5eebbdf2f48 
Resolving now.

**Review by justinlee-aerospike — APPROVED** [review]
changes look good

**Review by artanderson — APPROVED** [review]
Haven’t had the chance to test, but the changes look good to me.

**Review by justinlee-aerospike — APPROVED** [review]
changes look good.  eslint doesn't seem to do the formatting well.  Let's look into using prettier to format in the future.

**Pull request merged**